prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts

```diff
@@ -112,6 +114,12 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
     // Store current status as it will be truncated for initial enrolment
     this.currentStatus = enrolment.currentStatus.statusCode;
 
+    if (enrolment.expiryDate) {
+      const expiryMoment = moment(enrolment.expiryDate);
+      this.expiryDate = expiryMoment.isAfter(moment.now())
+        ? expiryMoment.format('MMMM Do, YYYY') : null;
```

Comment:
The ticket doesn't seem to specify, but would we want to hide the day it expired even if it has expired?